scripts/link_breakpoints

```diff
@@ -256,9 +260,10 @@ event_link_df = bind_rows(
   mutate(
     max_supporting_fragment_count = max(supporting_fragment_count),
     min_supporting_fragment_count = min(supporting_fragment_count),
-    hasPolyA=any(hasPolyA)
-    ) %>%
+    hasPolyA=any(hasPolyA),
+    sum_qual = if(any(has_pair, na.rm = TRUE)) { sum(QUAL, na.rm = TRUE) } else { 0 }) %>%
```

Comment:
Is the sum happening on the variants that were linked before the script? It probably should not, the qualities are effectively aggregated there in gridss

Reply:
oh you are right,
 I had this logic, but I don't find that. I will add that, thanks

Reply:
I have added that check tot he code, but it does not have any influence. All the variants in that table are without MATEID, so I think we will get the same results